enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -158,7 +160,7 @@
                   "for users that have already been authenticated successfully. A user can be authenticated against " +
                   "an existing cache entry (instead of via an LDAP server) as long as it is alive " +
                   "(see `dbms.security.auth_cache_ttl`).\n" +
-                  "An important consequence of setting this to `true` than needs to be well understood, is that " +
+                  "An important consequence of setting this to `true` that needs to be well understood, is that " +
```

Comment:
I think "that needs to be well understood" is redundant and could be removed all together